config/routes.rb

```diff
@@ -0,0 +1,4 @@
+Rails.application.routes.draw do
+  resources :users
```

Comment:
Sorry but the **routes** are restricted.
Kindly read it [here](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_route1.png)